subgraphs/aave-v2/src/common/constants.ts

```diff
@@ -137,22 +118,17 @@ export const MS_PER_YEAR = DAYS_PER_YEAR.times(new BigDecimal(BigInt.fromI32(24
 
 // This is the LendingPoolAddressesProviderRegistry contract address
 export const PROTOCOL_ADDRESS = "0x52D306e36E3B6B02c153d0266ff0f85d18BCD413";
-
 export const USDC_TOKEN_ADDRESS = "0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48";
-
 export const PRICE_ORACLE_ADDRESS = "0xa50ba011c48153de246e5192c8f9258a2ba79ca9";
```

Comment:
Instead of hardcoding this we could call `getPriceOracle()` when the protocol is created

Reply:
yeah, we are calling this [contract function](https://github.com/messari/subgraphs/pull/437/files#diff-5f5d7397e42bc85f6ccad939668677e03a5c641f711fd574d5a61f62c477f272R104-R106) when we initialize the context, this constant is just the default value if the RPC call fails.